Spigot-Server-Patches/0140-Delay-Chunk-Unloads-based-on-Player-Movement.patch

```diff
@@ -70,8 +75,26 @@ index c7af8da61..0adfcaa8b 100644
  
              this.chunkLoader.b();
          }
+diff --git a/src/main/java/net/minecraft/server/Entity.java b/src/main/java/net/minecraft/server/Entity.java
+index 13dba403..d8ae0327 100644
+--- a/src/main/java/net/minecraft/server/Entity.java
++++ b/src/main/java/net/minecraft/server/Entity.java
+@@ -330,6 +330,13 @@ public abstract class Entity implements ICommandListener {
+     }
+ 
+     public void B_() {
++        // Paper start - Skip ticking in chunks scheduled for unload
```

Comment:
Please move this check to checkIfActive instead on ActivationRange, where we already have a chunk reference.